boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -51,6 +51,23 @@
 			};
 		};
 
+		etr_buf_ram0x_region: memory@2f0be000 {
+			/* TODO to be reworked when SDFW with STM support is released.*/
```

Comment:
fyi: This depends on new UICR Trace fields, these won't come before the for the sdfw stm support so we also just rely on fixed position for the time being.